api/response/schema_query.py

```diff
@@ -194,9 +197,44 @@ def resolve_all_room_responses(
             has_next = paginated_response.has_next
             pages = paginated_response.pages
             query_total = paginated_response.query_total
-            return PaginatedResponses(responses=current_page,
-                                      has_previous=has_previous,
-                                      has_next=has_next,
-                                      query_total=query_total,
-                                      pages=pages)
+            return PaginatedResponses(
+                responses=current_page,
+                has_previous=has_previous,
+                has_next=has_next,
+                query_total=query_total,
+                pages=pages)
         return PaginatedResponses(responses=responses)
+
+
+class HandleResponse(graphene.Mutation):
```

Comment:
Use a more specific class name and handle this mutation class in the schema file

Reply:
Noted @Bonifase . Is UpdateResponse a good name? Because I had use MarkResponse and was advised to change it